android/sample/src/debug/java/com/facebook/flipper/sample/ExampleActions.java

```diff
@@ -22,66 +23,95 @@
 
 public final class ExampleActions {
 
-  public static void sendPostRequest(OkHttpClient client) {
-    final RequestBody formBody =
-        new FormBody.Builder().add("app", "Flipper").add("remarks", "Its awesome").build();
-
-    final Request request =
-        new Request.Builder()
-            .url("https://demo9512366.mockable.io/SonarPost")
-            .post(formBody)
-            .build();
-
-    client
-        .newCall(request)
-        .enqueue(
-            new Callback() {
-              @Override
-              public void onFailure(final Call call, final IOException e) {
-                e.printStackTrace();
-                Log.d("Flipper", e.getMessage());
-              }
-
-              @Override
-              public void onResponse(final Call call, final Response response) throws IOException {
-                if (response.isSuccessful()) {
-                  Log.d("Flipper", response.body().string());
-                } else {
-                  Log.d("Flipper", "not successful");
-                }
-              }
-            });
-  }
-
-  public static void sendGetRequest(OkHttpClient client) {
-    final Request request =
-        new Request.Builder().url("https://api.github.com/repos/facebook/yoga").get().build();
-    client
-        .newCall(request)
-        .enqueue(
-            new Callback() {
-              @Override
-              public void onFailure(final Call call, final IOException e) {
-                e.printStackTrace();
-                Log.d("Flipper", e.getMessage());
-              }
-
-              @Override
-              public void onResponse(final Call call, final Response response) throws IOException {
-                if (response.isSuccessful()) {
-                  Log.d("Flipper", response.body().string());
-                } else {
-                  Log.d("Flipper", "not successful");
-                }
-              }
-            });
-  }
-
-  public static void sendNotification() {
-    final FlipperClient client = AndroidFlipperClient.getInstanceIfInitialized();
-    if (client != null) {
-      final ExampleFlipperPlugin plugin = client.getPluginByClass(ExampleFlipperPlugin.class);
-      plugin.triggerNotification();
+   public static void sendPostRequest() {
```

Comment:
There is still a bunch of whitespace changed. Do you think you could try to adapt that to the current formatting style?